CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ All notable changes to this project will be documented in this file.
 ### Added
 - Support for a yaml settings file to collect and propagate metadata for CST beam files.
 
+### Fixed
+- Combining UVData objects to overwrite invalid "dummy data" no longer produces errors.
```

Comment:
maybe expand this to say something more about the case where people are likely to have run into this, where they have input files that don't have any dummy data (so they might not realize this fixes a bug they care about)

Reply:
maybe "fixes a bug that occurred when combining data files across multiple axes"?

Reply:
Updated.